docs/robots/jackal/user_manual_jackal.mdx

```diff
@@ -1,12 +1,358 @@
+|  <img src="/img/robot_images/dingo_images/icon-wifi.png" width="50"/>   | **Wi-Fi indicator**: LED is solid green when Wi-Fi is connected and LED is off otherwise                                                                                                                                                                                                                                                                                                                                |
+| <img src="/img/robot_images/dingo_images/icon-battery.png" width="50"/> | **Battery indicator**: LED is solid green when the battery level is ok, solid yellow when the battery level is getting low (should be charged soon), and solid red when the battery level is critical (should be charged immediately)                                                                                                                                                                                   |
+|  <img src="/img/robot_images/dingo_images/icon-power.png" width="50"/>  | **System Power button**: LED is solid blue when the system powered on and running and the LED is flashing blue during the shutdown process; when the system is powered off, pressing the power button begins the power on process; when the system is powered on, a quick press of the power button begins a graceful shutdown and a prolonged press of the power button triggers and immediate power off of the system |
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/dingo_images/hmi.png"
+      width="300"
+    />
+    <figcaption>HMI Panel</figcaption>
+  </figure>
+</center>
+
+#### Payloads
+
+Additional payloads such IMUs, lidars, cameras, GPS, manipulators and many more can be integrated
+with Jackal, either as part of your Jackal purchase or installed after receiving your Jackal.
+You can find a list of supported packages and related documentation for payloads in this
+documention portal, under the [Computers](/docs/computers), [Sensors](/docs/sensors), [Manipulators](/docs/manipulators),
+and [Accessories](/docs/accessories) sections. In addition, refer to [Jackal Integration](/docs/robots/jackal/integration_jackal) for
+details on how to integrate your own payload or how to integrate a payload kit from Clearpath Robotics.
+Each payload kit that ships from Clearpath Robotics comes with a custom mounting bracket and cabling for
+easy attachment to Jackal.
+
+#### Interior Features
+
+To access Jackal's interior, actuate the latches under the front end of the lid, on the opposite end from
+the HMI. When you lift the lid, you will see Jackal's onboard Li-Ion battery pack, and its two connectors.
```

Comment:
Suggest changing Li-Ion refereces to 'lithium'

Reply:
Updated for all robots